### PR TITLE
Tanstack Table Core. Subproject

### PR DESCRIPTION
**Pull request by aerialist7**
cc @turansky

### CODE REVIEW

File: kotlin-tanstack-table-core/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+plugins {
+    kotlin("js")
+    `kotlin-conventions`
+    `publish-conventions`
+}
+
+dependencies {
+    api(project(":kotlin-react-core"))
```
Comment (turansky) [inline kotlin-tanstack-table-core/build.gradle.kts:8]: It's react free artifact

File: kotlin-wrappers-bom/build.gradle.kts [hunk 2]
```diff
@@ -33,6 +33,7 @@ dependencies {
         api(project(":kotlin-ring-ui"))
         api(project(":kotlin-styled"))
         api(project(":kotlin-styled-next"))
+        api(project(":kotlin-tanstack-table-core"))
```
Comment (turansky) [inline kotlin-wrappers-bom/build.gradle.kts:36]: Is it in alphabet order?

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-tanstack-table-core/build.gradle.kts (hunk 1 above)
Comment (aerialist7) [reply kotlin-tanstack-table-core/build.gradle.kts]: Fixed

File: kotlin-wrappers-bom/build.gradle.kts (hunk 2 above)
Comment (aerialist7) [reply kotlin-wrappers-bom/build.gradle.kts:36]: Will fix it in next PR, because now it will produce merge conflict

**Review by turansky — APPROVED** [review]

**Pull request merged**